.travis.yml

```diff
@@ -80,6 +83,22 @@ install:
 script:
   - python -c "import pyproj; pyproj.Proj(init='epsg:4269')"
   - py.test --cov-report term-missing --cov=pyproj -v -s
+  # Building and uploading docs with doctr
+  - set -e
+  - |
+    if [ "$DOC" ]; then
+      pip install sphinx sphinx_rtd_theme
+      cd docs
+      make html
+      cd ..
```

Comment:
What are your thoughts on building the docs outside of the `if` and just be part of the tests?

Also, what are your thoughts on deploying docs in the `after_success` section?